Bonsai.Emergent/Properties/launchSettings.json

```
@@ -0,0 +1,10 @@
+{
+  "profiles": {
+    "Bonsai": {
+      "commandName": "Executable",
+      "executablePath": "C:\\Users\\Phields01\\AppData\\Local\\Bonsai\\Bonsai.exe",
```

```suggestion
      "executablePath": "$(registry:HKEY_CURRENT_USER\\Software\\Bonsai Foundation\\Bonsai@InstallDir)Bonsai.exe",
```

Can we replace this with the known registry key? We will replace it with a local environment anyway, but for now it would let us avoid merging paths to local test environments.